Sources/SimplenoteSearch/Query/SearchQuery.swift

```diff
@@ -77,3 +88,20 @@ public final class SearchQuery: NSObject {
         return items == object.items
     }
 }
+
+public struct SearchQuerySettings {
+    public let tagsKeyword: String
+    public let localizedTagKeyword: String
+    
+    public init(tagsKeyword: String, localizedTagKeyword: String) {
+        self.tagsKeyword = tagsKeyword
+        self.localizedTagKeyword = localizedTagKeyword
+    }
+}
+
+public extension SearchQuerySettings {
+    static var shared: SearchQuerySettings {
+        SearchQuerySettings(tagsKeyword: "tag:",
+                            localizedTagKeyword: NSLocalizedString("tag:", comment: "Search Operator for tags. Please preserve the semicolons when translating!"))
+    }
+}
```

Comment:
Let's try to drop the singleton!  Global state tends to be troublesome, for a myriad of reasons.

**Plus** having `NSLocalizedString("tag:"` here **will not** cause GlotPress to ingest this string, since this specific code lives in the SimplenoteSearch repository.

How about if `SearchQuery` expected a `SearchQuerySettings` instance in its initializer?
Then, in SN side of things you could do:

```
public extension SearchQuerySettings {
     static var default: SearchQuerySettings {
         SearchQuerySettings(tagsKeyword: "tag:",
                             localizedTagKeyword: NSLocalizedString("tag:", comment: "..."))
     }
 }
```